cmd/internal/flowmode/cmd_run.go

```diff
@@ -83,6 +88,12 @@ depending on the nature of the reload error.
 		},
 	}
 
+	cmd.Flags().
+		BoolVar(&r.configAutoReload, "config.auto-reload", r.configAutoReload, "Automatically reload the config file when it changes on disk")
+	cmd.Flags().
+		BoolVar(&r.configForcePolling, "config.force-polling", r.configForcePolling, "Default to using only polling instead of filesystem events for auto-reloading")
+	cmd.Flags().
+		DurationVar(&r.configPollInterval, "config.poll-interval", r.configPollInterval, "Interval at which to poll the config file for changes when polling is used")
```

Comment:
I'm not sure about the design of these flags at the moment. 

I don't understand the value of `--config.force-polling`; from my perspective it seems like users would always want fsnotify + polling enabled at the same time, or neither. Do you have a use case in mind? If not, I'd suggest taking this flag out to keep the API surface smaller until we know we need it. 

I'd also suggest having the auto-reload and polling interval to be in the same flag namespace so they appear together in the documentation for easy discovery:

* `--config.auto-reload` to toggle whether auto-reloading is enabled (fsnotify + polling)
* `--config.auto-reload.poll-interval` to customize the frequency at which auto-reloading manually polls the file for changes. 

Does this make sense? What do you think?